sim/hunter/items.go

```diff
@@ -273,6 +277,46 @@ func init() {
 			}
 		})
 	})
+
+	core.NewItemEffect(MaelstromsWrath, func(a core.Agent) {
+		hunter := a.(HunterAgent).GetHunter()
+
+		if hunter.pet == nil {
+			return
+		}
+
+		hunter.pet.PseudoStats.DamageDealtMultiplier *= 1.02
```

Comment:
Following up on the comment above:

```suggestion
		if hunter.pet == nil {
			return
		}

		hunter.pet.PseudoStats.DamageDealtMultiplier *= 1.02
		
		if !hunter.talents.BestialWrath {
		  return
		}
		
    hunter.RegisterAura(core.Aura{
		  Label: "Improved Bestial Wrath",
		  OnInit: func(aura *core.Aura, sim *core.Simulation) {
		    hunter.bestialWrathPetAura.Duration += 3
		  }
		}))
```